### PR TITLE
Interaction between s3 and dataframe

### PR DESCRIPTION
**Pull request by jianshen92**

### CODE REVIEW

File: aws/s3.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    response = s3_client.get_object(Bucket=s3_bucket, Key=s3_key)
+    status = response.get("ResponseMetadata", {}).get("HTTPStatusCode")
+
+    if status == 200:
+        print(f"Successful S3 get_object response. {s3_bucket=}, {s3_key=}, {status=}")
+        df = pd.read_parquet(io.BytesIO(response.get("Body").read()))
+        return df
+
+    print(f"Unsuccessful S3 get_object response. {s3_bucket=}, {s3_key=}, {status=}")
+    return None
```
Comment (keatmin) [inline aws/s3.md:101]: Wondering why you opt to not do a return print and explicitly adding return None.

File: aws/s3.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        response = s3_client.put_object(
+            Bucket=s3_bucket,
+            Key=s3_key,
+            Body=parquet_buffer.getvalue(),
+        )
+
+        status = response.get("ResponseMetadata", {}).get("HTTPStatusCode")
+        if status == 200:
+            print(
+                f"Successful S3 put_object response. {s3_bucket=}, {s3_key=}, {status=}"
```
Comment (keatmin) [inline aws/s3.md:81]: Interesting, I have read about this but never really used it myself. Apparently, it's a `python3.8` f-string formatting function to do {variable=}

Comment (jianshen92) [reply aws/s3.md:81]: learnt from anthony gor gor. output of f"{s3_bucket=}" will be `s3_bucket=your_bucket`

Comment (keatmin) [reply aws/s3.md:81]: No wonder it was so familiar. I think I saw from Anthony too

File: aws/s3.md (hunk 1 above)
Comment (jianshen92) [reply aws/s3.md:101]: Dont have strong opinion. I was thinking return `None` is more explicit.

**Review by keatmin — APPROVED** [review]
LGTM

**Pull request merged**